README.md

```diff
@@ -1 +1,115 @@
-# vespa-clone-backend
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
- [ ] On your readme, kindly add a "Tests" section with specific instructions about how to run your tests.  That way, your app will be well-documented. 💪 

- [ ] There are two files with tests not yet implemented:

![image](https://user-images.githubusercontent.com/57421823/208405741-67f9006f-a43d-485c-b945-f9861a847b02.png)

These are reservations_helper_spec.rb and reservation_spec.rb.   Kindly either add some tests on those files or remove them.

- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍